src/tools/dev/scripts/bv_support/bv_hdf5.sh

```diff
@@ -619,9 +569,11 @@ function build_hdf5
         sz_dir="${VISITDIR}/szip/${SZIP_VERSION}/${VISITARCH}"
         cf_szip="--with-szlib=${sz_dir}"
     fi
-    info "Configuring HDF5 with ZLib support."
-    cf_zlib="--with-zlib=${zlib_dir}"
-    zlib_dir="${VISITDIR}/zlib/${ZLIB_VERSION}/${VISITARCH}"
+    cv_zlib=""
+    if [[ "$DO_ZLIB" == "yes" ]]; then
+        info "Configuring HDF5 with ZLib support."
+        cf_zlib="--with-zlib=\"${VISITDIR}/zlib/${ZLIB_VERSION}/${VISITARCH}\""
+    fi
```

Comment:
You used cv_zlib once, then cf_zlib. Based on other usage, seems like the 'cv_zlib' should be 'cf_zlib'

Reply:
Whoops...that is a bug but a harmless one in bash because any attempt to ref a non-existent variable just creates nothing. Will fix though. Thx for finding.